mula/scheduler/utils/cron.py

```diff
@@ -0,0 +1,8 @@
+from datetime import datetime, timezone
+
+from croniter import croniter
+
+
+def next_run(expression: str, start_time: datetime = datetime.now(timezone.utc)):
```

Comment:
Perhaps `calculate_from` or `after` or ... not sure

Reply:
perhaps use the croniter naming? `get_next`?

Reply:
You can't assign `datetime.now(timezone.utc)` in the signature, otherwise it will always have the same value (based on the module import time). The Pythonic way is to assign a `None` as default, and assign a new datetime in the function body if the passed value is indeed `None`.